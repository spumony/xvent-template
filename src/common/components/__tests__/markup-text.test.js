import React from 'react';
import { render } from '@testing-library/react';

import MarkupText from '../markup-text';

describe('MarkupText', () => {
  it('should render properly', () => {
    const { container } = render(<MarkupText content="<p>Hello world!</p>" />);

    expect(container.firstChild).toMatchSnapshot();
  });

  it('should render properly with different tag', () => {
    const { container } = render(<MarkupText content="<p>Hello world!</p>" tag="a" />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
