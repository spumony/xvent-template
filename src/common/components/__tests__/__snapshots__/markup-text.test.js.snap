// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MarkupText should render properly 1`] = `
<span>
  <p>
    Hello world!
  </p>
</span>
`;

exports[`MarkupText should render properly with different tag 1`] = `
<a>
  <p>
    Hello world!
  </p>
</a>
`;
